plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/metastore/glue/TestDeltaLakeGlueMetastore.java

```diff
@@ -76,23 +84,31 @@
 import static io.trino.plugin.hive.HiveTestUtils.HDFS_ENVIRONMENT;
 import static io.trino.plugin.hive.HiveType.HIVE_STRING;
 import static io.trino.plugin.hive.metastore.StorageFormat.fromHiveStorageFormat;
+import static io.trino.plugin.hive.metastore.glue.AwsSdkUtil.getPaginatedResults;
 import static io.trino.spi.security.PrincipalType.ROLE;
 import static io.trino.testing.TestingConnectorSession.SESSION;
 import static java.lang.String.format;
+import static java.lang.System.currentTimeMillis;
 import static java.util.Locale.ENGLISH;
+import static java.util.concurrent.TimeUnit.DAYS;
 import static org.apache.hadoop.hive.metastore.TableType.EXTERNAL_TABLE;
 import static org.apache.hadoop.hive.metastore.TableType.VIRTUAL_VIEW;
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.assertj.core.api.Assertions.assertThatThrownBy;
 
 public class TestDeltaLakeGlueMetastore
 {
+    private static final Logger log = Logger.get(TestDeltaLakeGlueMetastore.class);
+
+    private static final String TEST_DATABASE_NAME_PREFIX = "test_";
```

Comment:
By using this prefix will have the outcome that this test will delete glue databases belonging to other connectors as well (e.g. : iceberg).
To avoid doing the same logic for Iceberg too, can we place this logic on a shared location? Naive idea - a new module under `testing`.

Reply:
@findepi WDYT ?

Reply:
this is fine, let's just not introduce a new module for this